fake-files/output-expected/spraggins/spraggins.raster.json

```diff
@@ -1,9 +1,13 @@
 {
-  "schema_version": "0.0.1",
+  "schemaVersion": "0.0.2",
+  "renderLayers": [
+    "Spraggins MxIF", 
+    "Spraggins IMS"
```

Comment:
Not an issue currently, but these will need to be unique. Do you think we should generate ids?

Reply:
The other alternative would be to use indices.

Reply:
I think this is ok.  It's nice to have human readable stuff.

Reply:
Agreed, but these schemas aren't supposed to be created by hand. I worry that there could be naming conflicts in the future. After all, the names in `renderLayers` are the same as those for the images in the array. I think it should be a reference in someway, whether that is by index or id.

Reply:
I agree - I think in the future, once the image schema is fully fleshed out (i.e with initial values etc. in the process of integrating #100) we'll be able to handle this better.